ptp/c4u/c4u.go

```diff
@@ -44,15 +45,22 @@ var defaultConfig = &server.DynamicConfig{
 }
 
 // Run config generation once
-func Run(config *Config, rb *clock.RingBuffer) error {
+func Run(config *Config, rb *clock.RingBuffer, st stats.Stats) error {
+	defer st.Snapshot()
```

Comment:
I don't believe running `st.Snapshot()` every second is what you want... Usually run of `st.Snapshot()` correlates with the collectors interval (aka once a minute).